src/client/styles/Grid.ts

```diff
@@ -80,15 +114,28 @@ export const gridRow = css`
   display: -ms-grid;
   display: grid;
   width: 100%;
-  column-gap: ${px(space[5])};
 
-  -ms-grid-columns: \(${COLUMN_WIDTH.MOBILE}\)\[${COLUMNS.MOBILE}\];
-  grid-template-columns: repeat(${COLUMNS.MOBILE}, ${COLUMN_WIDTH.MOBILE});
-  padding-left: ${px(SPACING.MOBILE)};
-  padding-right: ${px(SPACING.MOBILE)};
```

Comment:
These particular css lines were generated already using `generateGridRowCss` and `MOBILE`